packages/clay-badge/package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "@clayui/clay-badge",
```

Comment:
hey @bryceosterhaus as we now have the scope, I think it is no longer necessary to add `clay-`, just leaving `@clayui/badge`.